### ISSUE TITLE
[Accessibility] Screen reader doesn't announce item source in suggestions list for autocomplete

### ISSUE THREAD

**Issue by rafa11rodrigues** [microsoft/vscode#166073]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.73.1
- OS Version: Windows 10 and 11

Steps to Reproduce:

1. Open VS Code while running a screen reader
2. Create a project with a well-known structure and many folders (e.g. an Angular project, a Node project or a Java project)
3. In two or more different folders, create two or more entities with the same name or prefix. By entity I mean a class, function, constant, anything which will be imported somewhere else.
4. In another file of the project, start typing the entity name and wait for the suggestions list to show up.
5. Move up and down to listen to the suggestions. Note that only the entity name is spoken, but you don't know where they come from, even though this information is visually present at the right edge of the suggestion entry.

It often happens when your project has a class with the same name of a class you defined in your project. Sometimes you want to use your own class, sometimes you want to use the third party one. Then when you are typing and wants to autocomplete the class name to autoimport it, you don't know which one is the right because the screen reader doesn't say anything.

Fun fact: sometimes it works! Really, it's very random, sometimes the source is announced, but it's almost never...

**Comment by VSCodeTriageBot** [comment]
This feature request is now a candidate for our backlog. The community has 60 days to [upvote](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) the issue. If it receives 20 upvotes we will move it to our backlog. If not, we will close it. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
This feature request has not yet received the 20 community [upvotes](https://github.com/microsoft/vscode/wiki/Issues-Triaging#up-voting-a-feature-request) it takes to make to our backlog. 10 days to go. To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!

**Comment by VSCodeTriageBot** [comment]
:slightly_frowning_face: In the last 60 days, this feature request has received less than 20 community upvotes and we closed it. Still a big Thank You to you for taking the time to create this issue! To learn more about how we handle feature requests, please see our [documentation](https://aka.ms/vscode-issue-lifecycle).

Happy Coding!
